projects/batfish-common-protocol/src/test/java/org/batfish/datamodel/StaticRouteTest.java

```diff
@@ -0,0 +1,58 @@
+package org.batfish.datamodel;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/** Tests for {@link StaticRoute} */
+public class StaticRouteTest {
+
+   StaticRoute.Builder _srBuilder;
+
+   @Test public void checkAllAttrs() {
```

Comment:
It looks to me like you have not properly set up your google-java-format plugin. Please do so and reformat all changed files. (See https://github.com/batfish/batfish/wiki/IntelliJ)